crates/ee-acct-runtime/src/update_processing.rs

```diff
@@ -318,10 +319,17 @@ pub fn apply_final_update_changes(
 }
 
 fn verify_acct_state_matches(
-    _astate: &EeAccountState,
-    _exp_new_state: &Hash,
+    astate: &EeAccountState,
+    exp_new_state: &Hash,
 ) -> Result<(), EnvError> {
-    // TODO use SSZ hash_tree_root
+    // Compute SSZ tree_hash_root
+    // REVIEW: should we use Sha256Hasher or Sha256MerkleHasher?
+    let computed_root = TreeHash::<Sha256Hasher>::tree_hash_root(astate);
```

Comment:
I think you can simplify it since `Sha256Hasher` is the default in `ssz-gen`'s `tree_hash` crate:

```suggestion
    let computed_root = astate.tree_hash_root()
```

The `Sha256MerkleHasher` is used internally by `TreeHash` implementations when they need to merkleize collections (lists, vectors, containers with multiple fields). You never pass it as a type parameter to `TreeHash`.